apps/nowcasting-app/constant.ts

```diff
@@ -1,4 +1,4 @@
-export const API_PREFIX = "https://api-dev.nowcasting.io/v0";
+export const API_PREFIX = "https://api-pro.nowcasting.io/v0";
```

Comment:
what's the api-pro, @peterdudfield?

Reply:
Oh good point @rachel-labri-tipton, is there any reason this is not an env var if it's just switching between the dev/prod endpoints @peterdudfield?

Reply:
we should use ENV VAR. and development should definately be dev. 
Ill make an issue for this

Reply:
https://github.com/openclimatefix/nowcasting/issues/263